code/modules/mob_spawn/corpses/mining_corpses.dm

```diff
@@ -222,6 +254,30 @@
 	if(prob(50))
 		neck = /obj/item/bedsheet/rd/royal_cape
 
+/datum/outfit/consumed_ice_settler
+	name = "Legion-Consumed Settler"
+	suit = /obj/item/clothing/suit/hooded/wintercoat
+	shoes = /obj/item/clothing/shoes/winterboots
+	mask = /obj/item/clothing/mask/breath
```

Comment:
Hey I think the reason you can't put anything in this guy's pockets is that he isn't wearing any pants